app/scripts/modules/core/task/tasks.html

```diff
@@ -149,6 +149,11 @@ <h4 ng-if="!viewState.nameFilter && !viewState.taskStateFilter">We couldn't find
                 <strong>Exception:</strong> {{task.failureMessage}}
               </td>
             </tr>
+            <tr ng-if="tasks.isExpanded(task.id) && task.getValueFor('comments')" style="background-color: white;">
```

Comment:
Might as well move the inline style to the CSS, since you're already making changes to the LESS file.